lib/player.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'dealer'
+require_relative 'blackjack_score'
+
+class Player
+
+  def get_initial_hand
+    dealer = Dealer.new
+    player_one_hand = dealer.deal_initial_hand
+  end
```

Comment:
You've got a circular dependency here... The Player depends on the Dealer and the Dealer depends on the Player.  This is an unhappy situation.  I would like to see this refactored so it's only a one way dependency.

Utilising your `Hand` object and/or using dependency injection might help with this.